test-plone-5.1.x.cfg

```diff
@@ -6,14 +6,7 @@ extends =
 package-name = ftw.publisher.core
 
 auto-checkout =
-    plonetheme.onegov
-    ftw.simplelayout
 
 
-[branches]
-plonetheme.onegov = plone5
-ftw.simplelayout = plone-5.1.x
-
 [test]
 eggs +=
```

Comment:
✂️ test-section no needed, can be removed.

Reply:
👍